src/entrypoints/main/router/sparql/pages/sparql-page/index.tsx

```diff
@@ -4,29 +4,59 @@ import { compose } from 'redux';
 import Yasgui from '@triply/yasgui';
 import '@triply/yasgui/build/yasgui.min.css';
 
+import Link from '@fellesdatakatalog/link';
+
+import env from '../../../../../../env';
+
+import Translation from '../../../../../../components/translation';
+
 import SC from './styled';
 
+const { SPARQL_API_HOST } = env;
+
 interface Props {}
 
 const SparqlPage: FC<Props> = () => {
   const yasguiRef = useRef(null);
```

Comment:
Her kan du definere ref type for å slippe bruk av any videre
```suggestion
  const yasguiRef = useRef<TYPE DEFINISJON HER>(null);
```